towhee/functional/mixins/display.py

```diff
@@ -11,21 +11,75 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from typing import Iterable
-
+from typing import Iterable, Tuple
 from tabulate import tabulate
 
+from towhee.hparam import param_scope
+
+
+def _value_counts_callback(self):
+    def wrapper(_: str, index: Tuple[str], *arg, **kws):
+        # pylint: disable=import-outside-toplevel
+        # pylint: disable=unused-argument
+        import pandas as pd
+        columns = [[] for _ in index]
+        for x in self:
+            for i in range(len(index)):
+                columns[i].append(getattr(x, index[i]))
+
+        data = {}
+        for i in range(len(index)):
+            data[index[i]] = columns[i]
+        df = pd.DataFrame(data)
+        print(df.value_counts(**kws))
+
+    return wrapper
+
+
+def _plot_callback(self):
+    def wrapper(_: str, index: Tuple[str], *arg, **kws):
+        # pylint: disable=import-outside-toplevel
+        # pylint: disable=unused-argument
+        import pandas as pd
+        columns = [[] for _ in index]
+        for x in self:
```

Comment:
wont both of these callbacks consume the data of a streamed iterator? Will value_counts and plot only be supported as the last function of a DC chain like to_list()

Reply:
Yes, it will. Maybe I will add a check(transform to upstream) for it.

Reply:
We call this func with `param_scope().callholder`, so it will run in the workflow as usual functions.